.github/workflows/create-cli-deps-pr.yml

```diff
@@ -64,7 +64,6 @@ jobs:
           echo "Prepping CLI repo for release"
           cd cli
           git checkout "$npm_vtag"
-          make
           make release
```

Comment:
Removing this was due to the old default target being `make all` which just ran `make docs`. Docs are now built via `prepack` so when `scripts/release.sh` calls `pack` the docs will get built.